src/store/store.ts

```diff
@@ -542,6 +571,12 @@ export const useStore = defineStore('store', {
       )
     },
 
+    /** Whether all the amalgamation (regular) steps are valid. */
+    isAmalgamationRegularValid (): boolean {
+      // *** TODO: implement this
+      return false
+    },
```

Comment:
This is TODO for whomever adds components/data or works on the final "pull it all together" ticket.